eqcorrscan/core/match_filter.py

```diff
@@ -3360,6 +3386,170 @@ def normxcorr2(template, image):
     return ccc
 
 
+def multi_normxcorr(templates, stream, pads):
+    """
+    Compute the normalized cross-correlation of multiple templates with data.
+    :param templates: 2D Array of templates
+    :type templates: np.ndarray
+    :param stream: 1D array of continuous data
+    :type stream: np.ndarray
+    :param pads: List of ints of pad lengths in the same order as templates
+    :type pads: list
+
+    :return: np.ndarray
+    """
+    # TODO:: Try other fft methods: pyfftw?
+    import bottleneck
+    from scipy.signal.signaltools import _centered
+    from scipy.fftpack.helper import next_fast_len
+
+    # Generate a template mask
+    used_chans = ~np.isnan(templates).any(axis=1)
+    # Currently have to use float64 as bottleneck runs into issues with other
+    # types: https://github.com/kwgoodman/bottleneck/issues/164
+    stream = stream.astype(np.float64)
+    templates = templates.astype(np.float64)
+    template_length = templates.shape[1]
+    stream_length = len(stream)
+    fftshape = next_fast_len(template_length + stream_length - 1)
+    # Set up normalizers
+    stream_mean_array = bottleneck.move_mean(
+        stream, template_length)[template_length - 1:]
+    stream_std_array = bottleneck.move_std(
+        stream, template_length)[template_length - 1:]
+    # Normalize and flip the templates
+    norm = ((templates - templates.mean(axis=-1, keepdims=True)) / (
+        templates.std(axis=-1, keepdims=True) * template_length))
+    norm_sum = norm.sum(axis=-1, keepdims=True)
+    stream_fft = np.fft.rfft(stream, fftshape)
+    template_fft = np.fft.rfft(np.flip(norm, axis=-1), fftshape, axis=-1)
```

Comment:
Obspy 1.0.3 does not play nice on Windows, but the current master does, I'm going to keep the appveyor running the osbpy master and travis running the current release until the next obspy release when appveyor should revert to the current obspy release.  I think it comes down to their pinning of matplotlib...?